### ISSUE TITLE
Power Prompt doesn't output the same content as using the build-in nodes with the exact same input

### ISSUE THREAD

**Issue by stephantual** [rgthree/rgthree-comfy#83]
![ComfyUI_00006_](https://github.com/rgthree/rgthree-comfy/assets/1596070/8b41958a-706d-4ebf-8b83-de394fcd3a27)

Workflow attached - it's direct from 'comfy ui simple examples' so it's as simple and reproducible as possible.

I tried on more complex workflows - it generates slightly different outputs on the same seed, no lora invoved, just your bog standard generation. I used the display nodes and couldn't see anything wrong. I started digging into the code because I wanted to fix it but gave up after a couple hours. 

I also used the display nodes to compare outputs from the rgthree nodes to the input of the regular nodes and they are identical. I tried with and without quotes as well, to be safe.

TLDR: "something" (i suspect the hiddden params in the code regarding clipping, but i can't be 100% sure) is causing a non-deterministic output given the exact same conditionning and latent sources.

**Comment by rgthree** [comment]
There are two things:

1. The primitive positive & negative primitive nodes are not the same at the power prompts widgets; they have quotes and "\n\n" instead of actual newlines. That will have an impact on the generation.
2. More largely, though, the Negative Power Prompt doesn't have the clip_width/clip_height inputs connected, so it's falling back to non-SDXL Clip encoding. You'll see this in your comfyui terminal:
    
    ```
    [rgthree] SDXL Power Prompt - Simple / Negative: CLIP supplied, but 
       not CLIP_WIDTH and CLIP_HEIGHT. Text encoding will use standard
       encoding with prompt_g and prompt_l concatenated.
    ```
   You may have purposefully left them unconnected, since I see the negative native CLIPTextEncodeSDXL does not have them as inputs like the positive native CLIPTextEncodeSDXL does, but it still has the values as widgets, so they're being used there.

Here's the slight changes that should yield the same results:
![ComfyUI_00183_](https://github.com/rgthree/rgthree-comfy/assets/476360/ade95db6-cb05-4022-be6f-2f3df4525482)

**Comment by stephantual** [comment]
Thank you so much @rgthree - that's very kind of you to help me! i appreciate it!
